src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -24,6 +24,8 @@ public class MarkdownV2Renderer
 
         public int MaxSyntaxWidth { get; private set; }
 
+        private const string NewLine = "\r\n";
```

Comment:
Nice. We should probably do this generically. There is lot of separate references to `\r\n`

Reply:
Yeah, I just didn't want to introduce dependencies between classes to reference 2 well-known chars and creating a common helper seems like an overkill too.